presto-main/src/main/java/com/facebook/presto/event/QueryMonitor.java

```diff
@@ -260,6 +264,7 @@ public void queryCompletedEvent(QueryInfo queryInfo)
                         createOperatorStatistics(queryInfo),
                         createPlanStatistics(queryInfo.getPlanStatsAndCosts()),
                         historyBasedPlanStatisticsTracker.getQueryStats(queryInfo).values().stream().collect(toImmutableList()),
+                        getPlanHash(queryInfo.getPlanCanonicalInfo()),
```

Comment:
Another possibility is to reuse the hash information from `historyBasedPlanStatisticsTracker.getQueryStats(queryInfo).values().stream().collect(toImmutableList())` in the line above. However, this function only returns valid value when the query completes successfully, otherwise will return empty. 
Hence if I choose to reuse the information, I still need to make a change to enable it to return information even when query failed. Since the intention here is for logging only, I want to limit the change to not touch other parts of the code.